sqlglot/transforms.py

```diff
@@ -628,11 +628,30 @@ def eliminate_full_outer_join(expression: exp.Expression) -> exp.Expression:
             expression_copy = expression.copy()
             expression.set("limit", None)
             index, full_outer_join = full_outer_joins[0]
+
+            tables = (
+                full_outer_join.alias_or_name,
+                expression.args["from"].alias_or_name,
+            )
+            join_conditions = full_outer_join.args.get("on") or exp.and_(
+                *[
+                    exp.column(col, tables[0]).eq(exp.column(col, tables[1]))
+                    for col in full_outer_join.args.get("using")
+                ]
+            )
+
             full_outer_join.set("side", "left")
+            anti_join_clause = (
+                exp.Select(expressions=[exp.Literal(this=1, is_string=False)])
+                .from_(expression.args["from"])
+                .where(join_conditions)
+            )
             expression_copy.args["joins"][index].set("side", "right")
+            expression_copy.args["where"] = exp.Where(this=exp.Exists(this=anti_join_clause).not_())
```

Comment:
What if you already had a `WHERE` clause? This would overwrite it.

Reply:
Indeed, I'm converting it using the helper `where` since it's append by default